eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -119,8 +119,6 @@ jobs:
     - ${{ if in(parameters.platform, 'ios_arm64') }}:
       # split traffic for runtime-extra-platforms (which mostly runs on rolling builds)
```

Comment:
I have noticed that the `OSX.1015.Amd64.Iphone.Open` queue is being used for an arm64 device. It is possible that the architecture specified refers to the host instead of the device.

Reply:
Yes, the `Amd64` refers to the host machine, the `Iphone` part refers to a device that is attached to it and that one is `arm64` (it's a real iPhone device)